src/feeds/direct-inbox.feed.ts

```diff
@@ -4,6 +4,7 @@ import { DirectInboxFeedResponse, DirectInboxFeedResponseThreadsItem } from '../
 import { DirectThreadEntity } from '../entities';
 
 export class DirectInboxFeed extends Feed<DirectInboxFeedResponse, DirectInboxFeedResponseThreadsItem> {
+  limit: number;
```

Comment:
Set default value = 20 here

Reply:
@Expose() this property

Reply:
done

Reply:
@mjunior sweet update. Add this for following feed also